File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.util.Scanner;
 
public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		int[] num;
		int test = s.nextInt();
		
		for(int i = 0; i < test; i++) {
			int N = s.nextInt();
			num = new int[N];
			double sum = 0;

			for(int j = 0; j < N; j++) {
				int val = s.nextInt();
				num[j] = val;
				sum += val;
			}
			
			double aver = (sum / N) ;
			double count = 0;
			 
			for(int j = 0; j < N; j++) {
				if(num[j] > aver) {
					count++;
				}
			}
			System.out.printf("%.3f%%\n",(count/N)*100);

		}
	}
	
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cc
#include <iostream>
using namespace std;
int score[1000] = {0,};

int main () {
  cin.tie(NULL);
  ios_base::sync_with_stdio(false);

  int case_num;
  double result;
  cin >> case_num;

  for (int i = 0; i < case_num; i++) {
    int N, sum = 0, count = 0;
    cin >> N;
    
    for (int j = 0; j < N; j++) {
      cin >> score[j];
      sum += score[j];
    }
    
    double aver = sum / N;
    for (int j = 0; j < N; j++) {
      if (aver < score[j]) count++;
    }

    result = (double)count / N * 100;
    printf("%.3f%%\n", result);
  }

  return 0;
}